main.go

```diff
@@ -16,6 +16,25 @@ import (
 	_ "github.com/heroku/x/hmetrics/onload"
 )
 
+var (
+	downloadConcurrency  int
+	maxPayloadSizeInByte int64
+)
+
+func init() {
+	downloadConcurrency, _ = strconv.Atoi(os.Getenv("DOWNLOAD_CONCURRENCY"))
+	if downloadConcurrency <= 0 {
+		downloadConcurrency = 10
+		log.Println("DOWNLOAD_CONCURRENCY not specified. Setting it to", downloadConcurrency)
+	}
+
+	maxPayloadSizeInByte, _ = strconv.ParseInt(os.Getenv("MAX_PAYLOAD_SIZE_IN_BYTES"), 10, 64)
+	if maxPayloadSizeInByte <= 0 {
+		maxPayloadSizeInByte = 32 * 1024 * 1024 * 1024
```

Comment:
How about 32 << 30?

Reply:
I think expressing like it is currently in the change is more readable.

Reply:
@thamizhv nice way to do that if we are writing performance critical software. Would have more performant to calculate manually and assign a number. Point is to make it more readable and easy to change.